proto/tss/v1beta1/query.proto

```diff
@@ -147,3 +147,11 @@ message AssignableKeyRequest {
 }
 
 message AssignableKeyResponse { bool assignable = 1; }
+
+message ValidatorKeyRequest { string address = 1; }
```

Comment:
Why is address a string instead of bytes?

Reply:
Trying to match Cosmos SDK's use of primitive types for gRPC query inputs when possible.